src/shapes/textbox.class.js

```diff
@@ -271,7 +271,7 @@
       var wrapped = [], i;
       this.isWrapping = true;
       for (i = 0; i < lines.length; i++) {
-        wrapped = wrapped.concat(this._wrapLine(lines[i], i, desiredWidth));
+        wrapped.push.apply(wrapped, this._wrapLine(lines[i], i, desiredWidth));
```

Comment:
here there is a slight change.
before wrapped was always a new array, while now is mutated.
Is that something you want? or you dont care? i don't care.

Reply:
I don't care either.
What do think? Any side effects?

Reply:
if is not measurable in performance it does not matter.

Reply:
I thought it might boost perf. That's why.
Leaving it as is.